src/robot/parsing/model/statements.py

```diff
@@ -64,6 +68,22 @@ def from_tokens(cls, tokens):
                 return handlers[token.type](tokens)
         return EmptyLine(tokens)
 
+    @classmethod
+    def from_params(cls, *args, **kwargs):
+        """Create statement from passed parameters. Required and optional arguments
+        should match class properties. Values are used to create matching tokens.
```

Comment:
This doesn't follow the [general docstring conventions](https://www.python.org/dev/peps/pep-0257/) fully. Just using the first sentence as the short doc i.e. formatting like

```
"""Create statement from passed parameters.

Required and optional arguments should match class properties. Values are ...
```

would be a bit better.

More importantly, docs themselves are great!